lib/ping_pong/events/orders_order_cancel.js

```diff
@@ -12,14 +12,11 @@
  * @returns {Promise} p - resolves on completion
  */
 const onOrdersOrderCancel = async (instance = {}, order) => {
-  const { state = {}, h = {} } = instance
-  const { args = {}, orders = {}, gid } = state
+  const { h = {} } = instance
   const { emit, debug } = h
-  const { cancelDelay } = args
 
   debug('detected atomic cancelation, stopping...')
 
-  await emit('exec:order:cancel:all', gid, orders, cancelDelay)
```

Comment:
here we can use cancelDelay, but i think we should remove that feature all together...

Reply:
agreed